stack-7.10.yaml

```diff
@@ -1,15 +0,0 @@
-resolver: lts-6.16
```

Comment:
I removed the stack.yaml files because they appeared to be unmaintained (e.g. lts-6.16 instead of lts-6.35), some contained unnecessary `extra-deps`, and they are not used by CI. One can use `--resolver` instead of `--stack-yaml` with the added benefit of running the same command that CI runs.

Reply:
`--resolver` doesn't let you specify extra-deps, though